utils/check_deps.js

```diff
@@ -123,4 +123,12 @@ DEPS['src/trace/'] = ['src/utils/', 'src/client/**', 'src/server/**'];
 // Debug is a server plugin, nothing should depend on it.
 DEPS['src/debug/'] = ['src/utils/', 'src/generated/', 'src/server/**', 'src/debug/**'];
 
-checkDeps();
+// The service is a cross-cutting feature, and so it depends on a bunch of things.
+DEPS['src/remote/playwrightClient.ts'] = ['src/client/'];
```

Comment:
Might as well do a single DEPS entry for `src/remote/`.

Reply:
The client and server aren't separate? Or we don't care about the distinction?

Reply:
I think once we do care about the distinction, we'll put them into separate folders.